storage-tool.yml

```diff
@@ -306,10 +284,10 @@ services:
           - Access controlled by Brown Authentication and Security groups
       - name: storage
         class: 1 TB +
```

Comment:
what does storage of `1 TB +` mean in a world where the next class is `2 TB +`? what does the + indicate?

Reply:
@mcmcgrath13 : Excellent question!! The answer is not super satisfying... Both `1 TB +` and `2 TB +` don't really refer to ceilings. It's basically what a PI will get by default. 

The `+` indicates that there is the option for more beyond that `1 TB` or `2 TB`. In particular, for some it's an additional `10 TB` or an additional `20 TB` _per grant_. Or, it could also be the additional `TB` purchased at our current rate (i.e., $100/TB/yr). This isn't super easy thing to put in a table, so I guess we settled on `+`!!

Reply:
Maybe we should get rid of the question on "how much data do you have" then? as the 1 TB+ are still totally viable, just not all free (but neither are the 2 TB+ if you have 100 TB)